ifaces.go

```diff
@@ -1,22 +1,40 @@
 package cloudstorage
 
 import (
+	"io"
 	"os"
 	"time"
 )
 
 type Store interface {
```

Comment:
Our [godoc](https://godoc.org/github.com/lytics/cloudstorage#Store) would look a little nicer if this interface had a doc line and referenced some implementations since we are adding more.